src/config.rs

```diff
@@ -542,6 +553,7 @@ pub(crate) fn get_internal_config(config: CliConfig) -> SpotifydConfig {
         password,
         use_keyring: config.shared_config.use_keyring,
         cache,
+        audio_cache,
```

Comment:
The SpotifydStruct is only used internally. And since no other methods rely on the audio_cache flag, this can be removed.